modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/Model.scala

```diff
@@ -67,6 +67,7 @@ object Model {
     val breakpoint: Boolean
     val skip: Boolean
     val configStatus: Map[SystemName, ActionStatus]
+    val fileId: Option[dhs.ObsId]
```

Comment:
I'm not sure this is the right place, Perhaps it should be part of the `observeStatus`. WDYT?